### PR TITLE
Update kube-proxy addon. Mention skew-policy

### PR DESCRIPTION
**Pull request by orsenthil**
*Issue #, if available:*

* Update kube-proxy addon. Mention skew-policy

*Description of changes:*

* Updated the addons versions to the latest which will be available from 18-July-2022
* Mention about kube-proxy version skew policy

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by M00nF1sh — APPROVED** [review]
/lgtm
nit: shall we remove these whitespace changes?

File: doc_source/managing-kube-proxy.md [hunk 1]
```diff
@@ -5,19 +5,21 @@
...
+If you've added the `kube-proxy` Amazon EKS add\-on to your `1.19` or later cluster, you can manage it using the procedures in the [Updating the `kube-proxy` Amazon EKS add\-on](#updating-kube-proxy-eks-add-on) and [Removing the `kube-proxy` Amazon EKS add\-on](#removing-kube-proxy-eks-add-on) sections\. For more information about Amazon EKS add\-ons, see [Amazon EKS add\-ons](eks-add-ons.md)\.
 
 If you have not added the `kube-proxy` Amazon EKS add\-on, the `kube-proxy` self\-managed add\-on is still running on your cluster\. You can manually update the `kube-proxy` self\-managed add\-on using the procedure in [Updating the `kube-proxy` self\-managed add\-on](#updating-kube-proxy-add-on)\.
 
+For version compatiblity and skew policy, EKS adopts upsteam kubernetes [kube-proxy version skew policy](https://kubernetes.io/releases/version-skew-policy/#kube-proxy). That means, EKS version at minor version N can support upto N-2 minor versions of kube-proxy addon.
```
Comment (achevuru) [inline doc_source/managing-kube-proxy.md:21]: upsteam --> upstream

Comment (achevuru) [reply doc_source/managing-kube-proxy.md]: How about this minor rewording?  
"For version compatibility and skew policy, EKS adopts upstream kubernetes [version skew policy for kube-proxy](https://kubernetes.io/releases/version-skew-policy/#kube-proxy). So, EKS at minor version N can support upto N-2 minor versions of kube-proxy addon."

Comment (orsenthil) [reply doc_source/managing-kube-proxy.md]: Adopted with a minor change. Thank you.

File: doc_source/managing-kube-proxy.md [hunk 2]
```diff
@@ -5,19 +5,21 @@
...
+If you've added the `kube-proxy` Amazon EKS add\-on to your `1.19` or later cluster, you can manage it using the procedures in the [Updating the `kube-proxy` Amazon EKS add\-on](#updating-kube-proxy-eks-add-on) and [Removing the `kube-proxy` Amazon EKS add\-on](#removing-kube-proxy-eks-add-on) sections\. For more information about Amazon EKS add\-ons, see [Amazon EKS add\-ons](eks-add-ons.md)\.
 
 If you have not added the `kube-proxy` Amazon EKS add\-on, the `kube-proxy` self\-managed add\-on is still running on your cluster\. You can manually update the `kube-proxy` self\-managed add\-on using the procedure in [Updating the `kube-proxy` self\-managed add\-on](#updating-kube-proxy-add-on)\.
 
+For version compatiblity and skew policy, EKS adopts upstream kubernetes [kube-proxy version skew policy](https://kubernetes.io/releases/version-skew-policy/#kube-proxy). Accordingly, EKS at minor version N can support upto N-2 minor versions of kube-proxy addon.
```
Comment (achevuru) [inline doc_source/managing-kube-proxy.md:21]: compatiblity ---> compatibility

**Review by achevuru — APPROVED** [review]

**Pull request merged**